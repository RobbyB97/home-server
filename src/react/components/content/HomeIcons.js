import React from 'react'

import HomeIcon from './HomeIcon'


export default class HomeIcons extends React.Component {

    render() {
        return (
            <div className="homeIcons">
                {this.props.icons &&
                    this.props.icons.map((icon) => {
                        <HomeIcon {...icon} />
                    })
                }
            </div>
        )
    }
}