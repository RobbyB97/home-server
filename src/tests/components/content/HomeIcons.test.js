import React from 'react'
import {shallow} from 'enzyme'

import HomeIcons from '../../../react/components/content/HomeIcons'


test('Render HomeIcons without icons', () => {
    const wrapper = shallow(<HomeIcons />)

    expect(wrapper).toMatchSnapshot()
})