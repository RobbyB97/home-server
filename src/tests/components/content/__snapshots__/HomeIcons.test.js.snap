// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render HomeIcons without icons 1`] = `
<div
  className="homeIcons"
/>
`;
